proposals/governance/incubation_dd.md

```diff
@@ -61,6 +61,14 @@ KubeVirt advertises communications channels via https://kubevirt.io/community as
  * An OpenAPI definition as the only source of truth for KubeVirt’s API
  * https://github.com/kubevirt/kubevirt/pull/3349
 * [DONE] Stabilize or replace bridge network binding
+* [DONE] Disk hot add
+* [DONE] IPv6 support
+* [DONE] Device passthrough support
+* [DONE] Memory ballooning support
```

Comment:
We support the device for gathering memory statistics from the guest, but we don't support actual ballooning.